docs/using/tutorial/Vagrantfile.template

```diff
@@ -10,6 +10,41 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+$bootstrap_common = <<EOF
+mkdir -p /etc/flocker
+cp /home/vagrant/credentials/cluster.crt /etc/flocker/cluster.crt
+EOF
+
+$bootstrap_control = <<EOF
+cp /home/vagrant/credentials/control-172.16.255.250.crt /etc/flocker/control-service.crt
+cp /home/vagrant/credentials/control-172.16.255.250.key /etc/flocker/control-service.key
+systemctl enable flocker-control
+systemctl start flocker-control
+EOF
+
+$bootstrap_node1_cert = <<EOF
```

Comment:
As per the comment at https://github.com/ClusterHQ/flocker/pull/1466#issuecomment-108053101 I think that this should be done.